bitbots_bringup/config/bining.yaml

```diff
@@ -0,0 +1,3 @@
+decimation_x: 4
+decimation_y: 4
+interpolation:  3
```

Comment:
Is this file really supposed to be called `bining` and not  something else like `binding`?

Reply:
Oh there's a n missing. It should say `binning` which is a method to combine multiple pixels into a single one, to reduce resolution and noise.